src/build.d

```diff
@@ -1040,8 +1080,9 @@ auto isUpToDate(string[] targets, string[][] sources...)
     foreach (target; targets)
     {
         auto sourceTime = target.timeLastModified.ifThrown(SysTime.init);
+
         // if a target has no sources, it only needs to be built once
-        if (sources.empty || sources.length == 1 && sources.front.empty)
+        if (sources.empty || sources.length >= 1 && sources.front.empty)
```

Comment:
This isn't correct. See the line below.

Reply:
`sources` has a `length` of 2, but both are empty for the `cloneTools` dependency, but I don't know why yet.  Why is `sources` a `string[][]` instead of a `string[]`?

Reply:
> This isn't correct

What about now?